core/src/test/scala/org/apache/spark/scheduler/SchedulerIntegrationSuite.scala

```diff
@@ -518,10 +518,11 @@ class BasicSchedulerIntegrationSuite extends SchedulerIntegrationSuite[SingleCor
 
       // make sure the required map output is available
       task.stageId match {
-        case 1 => assertMapOutputAvailable(b)
-        case 3 => assertMapOutputAvailable(c)
         case 4 => assertMapOutputAvailable(d)
-        case _ => // no shuffle map input, nothing to check
+        case _ =>
+        // we can't check for the output for the two intermediate stages, unfortunately,
+        // b/c the stage numbering is non-deterministic, so stage number alone doesn't tell
+        // us what to check
```

Comment:
I thought about this, but decided it was kind of silly since you could be checking for something irrelevant, and its not a trivial change (you'd need to catch an exception from doing the wrong check).  Anyway just figured I'd leave it out for the hotfix.